selfdrive/car/ford/interface.py

```diff
@@ -50,11 +50,26 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.steerRatio = 16.8
       ret.mass = 2050
 
+    elif candidate == CAR.F_150_LIGHTNING_MK1:
+      ret.wheelbase = 3.69
+      ret.steerRatio = 17.0
+      ret.mass = 2729
+
+    elif candidate == CAR.MUSTANG_MACH_E_MK1:
+      ret.wheelbase = 2.984
+      ret.steerRatio = 14.6
+      ret.mass = 1959
+
     elif candidate == CAR.F_150_MK14:
       # required trim only on SuperCrew
       ret.wheelbase = 3.69
       ret.steerRatio = 17.0
-      ret.mass = 2000
+      ret.mass = 2728
+
+    elif candidate == CAR.F_150_LIGHTNING_MK1:
```

Comment:
I think block  from lines 53-56 are identical with  69-72.

Reply:
It is. I had plans to actually close this PR after I was finished working with shane, but I think I'm actually just gonna open a new github issue and copy the relevant info over to that so that info is there. I have a new (slightly) better branch to open a PR with after he and I can work through the fingerprint issue.